src/chunkserver/chunkserver.cpp

```diff
@@ -152,12 +151,12 @@ int ChunkServer::Run(int argc, char** argv) {
         if (!useChunkFilePoolAsWalPool) {
             FilePoolOptions walFilePoolOptions;
             InitWalFilePoolOptions(&conf, &walFilePoolOptions);
-            kWalFilePool = std::make_shared<FilePool>(fs);
-            LOG_IF(FATAL, false == kWalFilePool->Initialize(walFilePoolOptions))
+            walFilePool = std::make_shared<FilePool>(fs);
```

Comment:
What‘s the problem with global variables in this scenario？

Reply:
@SeanHai 

There are several reasons:
* Global variable is inelegant 
* When we stat the total number of WAL file, we need aggregate the number of WAL file in ervery copyset, so we need to know which copyset each `log storage` belongs to
* [See also](https://github.com/opencurve/curve/pull/309/files#diff-8a7f5ab58978d7cb1a50c7d167d34cacbcaeff88f246cc2117296285eb4eba23R191) for detail